docs/src/content/docs/reference/scripts/mcp-tools.mdx

```diff
@@ -46,6 +46,64 @@ See [MCP server](/genaiscript/reference/scripts/mcp-server) for more details.
 
 :::
 
+## CLI MCP Configuration
+
+### Using MCP configuration files
+
+You can also load MCP servers from a Claude format configuration file using the `--mcp-config` option when running scripts:
+
+```bash
+genaiscript run my-script --mcp-config .vscode/mcp.json
+```
+
+The configuration file uses the Claude MCP format:
+
+```json title="mcp.json"
+{
+    "servers": {
+        "filesystem": {
+            "type": "stdio",
+            "command": "npx",
+            "args": ["-y", "@modelcontextprotocol/server-filesystem", "${workspaceFolder}"],
+            "env": {
+                "DEBUG": "${env:DEBUG}"
+            }
+        },
+        "memory": {
+            "command": "npx",
+            "args": ["-y", "@modelcontextprotocol/server-memory"]
+        }
+    }
+}
+```
+
+### Environment Variable Interpolation
+
+The configuration file supports Claude environment variable interpolation syntax:
+
+- `${workspaceFolder}` - Resolves to the workspace folder (or the directory containing the config file)
+- `${env:VARIABLE_NAME}` - Resolves to the value of the environment variable `VARIABLE_NAME`
```

Comment:
@colpilot also support ${VARIABLE_NAME} to resolve a env variable (must be capitalized)